### PR TITLE
Copyright Added in the Footer Section.

### PR DESCRIPTION
**Pull request by its-VinayKumar**
**What kind of change does this PR introduce?**

**Issue Number:**
#573. Bug Resolved

**Screenshots/videos:**

**If relevant, did you update the documentation?**

**Summary**
Copyright implement in the Footer Section.

Screenshot (209)

### CODE REVIEW

**Review by benjagm — CHANGES_REQUESTED** [review]
It work great in desktop but not in mobile

Screenshot 2024-03-26 at 14 20 46

File: yarn.lock [hunk 1]
```diff
@@ -182,10 +182,10 @@
   resolved "https://registry.yarnpkg.com/@eslint-community/regexpp/-/regexpp-4.10.0.tgz#548f6de556857c8bb73bbee70c35dc82a2e74d63"
```
Comment (benjagm) [inline yarn.lock:182]: Let's not modify this file.

File: package.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
     "@typescript-eslint/eslint-plugin": "^6.10.0",
     "@typescript-eslint/parser": "^6.10.0",
     "autoprefixer": "^10.4.16",
-    "eslint": "8.53.0",
+    "eslint": "^8.57.0",
```
Comment (benjagm) [inline package.json:59]: Let's not modify this file.

**Review by benjagm — CHANGES_REQUESTED** [review]
It work great in desktop but not in mobile

Screenshot 2024-03-26 at 14 20 46

**Comment by its-VinayKumar** [comment]
@benjagm Ok, But the changes in package.json and yarn.lock are automatically done.

**Comment by its-VinayKumar** [comment]
Screenshot (214)
@benjagm Changes Updated. Please merge the file. :)

**Comment by DhairyaMajmudar** [comment]
> @benjagm Ok, But the changes in package.json and yarn.lock are automatically done.

Yeah, its because you are using `npm` instead of `yarn`. Discussion here will help you https://github.com/json-schema-org/website/pull/498

**Pull request closed without merging**